lib/mayaUsd/ufe/UsdUndoAddNewPrimCommand.cpp

```diff
@@ -48,7 +48,7 @@ UsdUndoAddNewPrimCommand::UsdUndoAddNewPrimCommand(const UsdSceneItem::Ptr& usdS
     _stage = usdSceneItem->prim().GetStage();
     if (_stage) {
         // Append the parent path and the requested name into a full ufe path.
-        _newUfePath = appendToPath(ufePath, name);
+        _newUfePath = appendToPath(ufePath, name + std::to_string(1));

```

Comment:
Added my earlier fix for MAYA-104858 here.

Reply:
Did this cause a test to fail to indicate the regression?  If not, we're missing a test...

Reply:
It looks not. This is the new class that @fowlertADSK introduced.  Agreed, we need a test for this.